src/applications/accredited-representative-portal/containers/POARequestsPage.jsx

```diff
@@ -1,17 +1,46 @@
 import React from 'react';
-
+import { useLoaderData, NavLink, Outlet } from 'react-router-dom';
 import DigitalSubmissionAlert from '../components/DigitalSubmissionAlert/DigitalSubmissionAlert';
-import POARequestsTableFetcher from '../components/POARequestsTableFetcher/POARequestsTableFetcher';
-import usePOARequests from '../hooks/usePOARequests';
 
 const POARequestsPage = () => {
+  const poaRequests = useLoaderData();
+
   return (
     <>
       <h1 data-testid="poa-requests-heading">Power of attorney requests</h1>
       <DigitalSubmissionAlert />
       <div className="poa-requests-page-table-container">
-        <h2 data-testid="poa-requests-table-heading">Requests</h2>
-        <POARequestsTableFetcher usePOARequests={usePOARequests} />
+        {poaRequests.length === 0 ? (
+          <p data-testid="poa-requests-table-fetcher-no-poa-requests">
+            No POA requests found
+          </p>
```

Comment:
Tabs should continue exist even when fetching poa requests returns no results.

Reply:
going forward can we write better tickets that reflect all the different types of views that a rep may see? I know we are doing a lot of things at once, but have noticed little things that are not included in some tickets.

Reply:
Yeah, we need better descriptions in tickets. Candi and I will work on refining some of our existing tickets further.

Reply:
updated so that the pending view is the default view